word-guess.rb

```diff
@@ -0,0 +1,97 @@
+
+require 'colorize'
+require 'Faker'
+class Game
+  attr_reader  :name
+ def initialize (name)
+   @name = name
+ end
+  def empty_array?(array)
+    array.each do |value|
+      return false if value != ""
+    end
+  end
+
+  def display_underlines(array_of_underlines)
+    print "Word: "
+    array_of_underlines.each do |underline|
+      print underline
+    end
+    puts
+  end
+
+  def input_incorrect?(user_guess, random_word)
+    special = "@#%!*1234567890-=+()"
+    if special.include?(user_guess)
+      return true
+    end
+  end
+
+  def find_place (random_word, user_guess, array_of_underlines)
+    random_word.each_with_index do |char, index|
+      if char == user_guess
+        array_of_underlines[index] = user_guess # put guessed letter in array of underlines
+        random_word[random_word.index(user_guess)] = ''
+      end
+    end
+  end
+
+  def play_game
+    stop_game = false
+    until stop_game
+      puts "Welcome to #{@name}! "
+      random_word = Faker::Color.color_name.split''
+      array_of_underlines = Array.new(random_word.length, "_ ")
+      display_underlines(array_of_underlines)
+      counter = 0 # counts wrong guesses, if 5 - user lose
+      puts "\n Enter a letter or a full word:"
+      flower = '
+    (@)(@)(@)(@)(@)
+      ,\,\,|,/,/,
+         _\|/_
+        |_____|
+         |   |
+         |___|'.colorize(:red)
+      stop_round = false # stop round of a game
```

Comment:
Could you think of a way to use the `break` and `next` statements we learned about in class to control the loop, instead of needing a separate variable?